README.rst

```diff
@@ -566,6 +566,10 @@ variable.
 		directory is searched after these. Thus, if this
 		option is not given, only the current directory is
 		searched. This has no effect when given in the config file.
+:configdump:	File to write the final configuration to. This includes
+		the results of all ``include'' directives, expansions
+		of environment variables and application of default
+		values - think ``gcc -E''.
```

Comment:
The options has a required argument. It's a bit confusing but I understand that making it optional is quite complex and probably not worth it. So I think it should just be documented, especially that `-` can be used for stdout.

Reply:
I somehow skipped over the first sentence while reading this... I shouldn't do reviews first thing in the morning :-).

And while `-` is quite common, I still think we should explicitly specify it. "Use ``-'' for stdout." or something like that.

Reply:
Added, and also a patch fixing the markup bug in includepath I'm responsible for.